app/src/main/java/org/jellyfin/androidtv/ui/playback/nextup/NextUpViewModel.kt

```diff
@@ -23,6 +26,7 @@ class NextUpViewModel(
 	val item: LiveData<NextUpItemData?> = _item
 	private val _state = MutableLiveData(NextUpState.INITIALIZED)
 	val state: LiveData<NextUpState> = _state
+	private val userPreferences by KoinJavaComponent.inject(UserPreferences::class.java)
```

Comment:
```suggestion
	private val userPreferences by inject<UserPreferences>()
```

Reply:
When I tried this it gave an error:

> No value passed for parameter 'clazz'

Removing `KoinJavaComponent.` works though:
`private val userPreferences by inject(UserPreferences::class.java)`

Reply:
That error means the import is wrong, try removing the import and have android studio import it again

Reply:
Android studio seems to only want to import the KoinJavaComponent one. I tried removing that import and manually adding `import org.koin.android.ext.android.inject` and `import org.koin.core.component.inject` but it won't use either of them :/

> Unresolved reference. 
> None of the following candidates is applicable because of receiver type mismatch:
> - public inline fun <reified T : Any> ComponentCallbacks.inject(qualifier: Qualifier? = ..., mode: LazyThreadSafetyMode = ..., noinline parameters: ParametersDefinition? /* = (() → DefinitionParameters)? */ = ...): Lazy<TypeVariable(T)> defined in org.koin.android.ext.android
> - public inline fun <reified T : Any> KoinComponent.inject(qualifier: Qualifier? = ..., mode: LazyThreadSafetyMode = ..., noinline parameters: ParametersDefinition? /* = (() → DefinitionParameters)? */ = ...): Lazy<TypeVariable(T)> defined in org.koin.core.component